.github/workflows/test-check.yml

```diff
@@ -0,0 +1,78 @@
+name: Test Checker
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release/*
+
+jobs:
+  test_check:
+    name: Check test results
+    runs-on: ubuntu-latest
+    steps:
+      - name: Wait for build to succeed ( ubuntu )
+        uses: tomchv/wait-my-workflow@v1.1.0
+        id: wait-for-build-ubu
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: "Tests & Build (ubuntu-latest)"
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          timeoutSeconds: 7200
+
+      - name: Make sure the build was a success ( ubuntu )
+        if: steps.wait-for-build-ubu.outputs.conclusion != 'success'
+        run: echo buils not success && false
```

Comment:
```suggestion
        run: echo "Ubuntu build has failed" && false
```

Reply:
this is outdated, the name of the OS is in the title and the status is now always printed.